services/bookables-api/README.md

```diff
@@ -0,0 +1,99 @@
+# HELSINGBORG IO SLS BOOKABLES SERVICE
+
+## Purpose
+
+The Bookables Service's purpose is to provide the app with data related to services that can be booked at Helsingborg Stad.
+
+## Description
+
+The Bookables Service is an API that returns information about the bookable services at Helsingborg Stad. Currently it retrieves the data from the parameter store, for ease of deployment. In the future this could be developed further into a platform with its own dynamo.
```

Comment:
I don't think that we should write what the service could be. Instead only describe how the service is at this particular moment.

Reply:
Good idea, I have updated the README.